src/utility/copy.cpp

```diff
@@ -73,22 +60,25 @@ CreateRelationCopyString(ParseState *pstate, CopyStmt *copy_stmt) {
 		                RelationGetRelationName(rel))));
 	}
 
-	std::string relation_copy = pgduckdb_relation_name(relid);
-	if (copy_stmt->attlist) {
-		ListCell *lc;
-		relation_copy += "(";
-		bool first = true;
-		foreach (lc, copy_stmt->attlist) {
-			if (!first) {
-				relation_copy += ", ";
-			}
+	appendStringInfoString(info, pgduckdb_relation_name(relid));
+	if (!copy_stmt->attlist) {
+		return;
+	}
+
+	ListCell *lc;
+	appendStringInfo(info, "(");
+	bool first = true;
+	foreach (lc, copy_stmt->attlist) {
```

Comment:
Let's change this to use the modern `foreach` macros instead while we're at it.
```suggestion
	foreach_node (String, attr, copy_stmt->attlist) {
```